api/hyp3_api/__init__.py

```diff
@@ -0,0 +1,8 @@
+import boto3
+import connexion
+
+connexion_app = connexion.App(__name__)
+BATCH_CLIENT = boto3.client('batch')
+
+from hyp3_api import auth, handlers  # noqa Has to be at end of file or will cause circular import
```

Comment:
maybe `# noqa (suppress linting, has to be at end of file or will cause circular import)`